.circleci/config.yml

```diff
@@ -176,22 +176,30 @@ commands:
             echo "Canceling workflow as previous step resulted in failure."
             echo "To execute all checks locally, please run yarn ci-tests"
             curl -X POST --header "Content-Type: application/json" "https://circleci.com/api/v2/workflow/${CIRCLE_WORKFLOW_ID}/cancel?circle-token=${WORKFLOW_CANCELER}"
+  list-recent-merges:
```

Comment:
@IanVS I guess this should not be part of this PR right?

Reply:
WTF, ugh.